### PR TITLE
native vm image building!

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
NONE
```

see docs

### CODE REVIEW

**Comment by jim-minter** [comment]
/hold
still at least a few dns issues with the built image

**Comment by mjudeikis** [comment]
And Jim did it again! O_O 
This looks awesome!

**Comment by jim-minter** [comment]
/hold cancel

ready to merge if someone will lgtm it

File: pkg/vmimage/data/script.sh [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+cat >client-cert.pem <<'EOF'
+{{ .Extra.Builder.ClientCert | CertAsBytes | String }}
+EOF
+
+cat >client-key.pem <<'EOF'
+{{ .Extra.Builder.ClientKey | PrivateKeyAsBytes | String }}
+EOF
+
+go get github.com/jim-minter/tlsproxy
```
Comment (mjudeikis) [inline pkg/vmimage/data/script.sh]: follow-up for this

**Pull request merged**